bundles/distribution/src/main/assembly/eclipselink.zip.xml

```diff
@@ -24,57 +24,37 @@
     <dependencySets>
         <dependencySet>
             <outputDirectory>eclipselink/jlib/</outputDirectory>
-            <outputFileNameMapping>${artifact.artifactId}.${artifact.extension}</outputFileNameMapping>
+            <outputFileNameMapping>${artifact.artifactId}${dashClassifier?}.${artifact.extension}</outputFileNameMapping>
             <includes>
                 <include>${project.parent.groupId}:eclipselink</include>
```

Comment:
@rfelcman this line here covers eclipselink.jar, eclipselink-sources.jar and eclipselink-javadoc.jar as long as they are defined as project dependencies